rpa-builder/6.0/modules/ROOT/pages/analyzing-problems-with-analysis-packages.adoc

```diff
@@ -0,0 +1,49 @@
+= Analyzing Problems with Analysis Packages
+
+After the business process runs locally in RPA Builder, you can upload it to RPA Manager, move it to the test phase, run it on an RPA Bot, and start a long-term test. We recommend a test period of three days before using the business process productively. If menu runs contain errors, RPA Manager generates Analysis Packages that you can download.
```

Comment:
```suggestion
After the business process runs locally in RPA Builder, you can upload it to RPA Manager, move it to the test phase, run it on an RPA Bot, and start a long-term test. Run tests for at least three days before using the business process productively. If menu runs contain errors, RPA Manager generates Analysis Packages that you can download.
```

Reply:
this seems like a nit to me, but I think the style guide says not use constructions like "We recommend..."